fabtests/pytest/common.py

```diff
@@ -217,7 +220,7 @@ def prepare_base_command(self, command_type, executable,
             if not has_cuda(self._cmdline_args.client_id):
                 pytest.skip("no cuda device")
                 return
-            if not has_hmem_support(self._cmdline_args, "client", self._cmdline_args.client_id):
```

Comment:
Why run the same line twice? There's no difference between 210 and 220

Reply:
It's checking on client and server both.

Reply:
But it seems with unit test, both check are running on the same node. Need to fix.

Reply:
Construct the command explicitly in newest revision.